src/test/java/seedu/address/logic/parser/CommandParserTestUtil.java

```diff
@@ -16,6 +16,7 @@
      */
     public static void assertParseSuccess(Parser parser, String userInput, Command expectedCommand) {
         try {
+
```

Comment:
You can remove this line as well while you are reverting the changes to lchokhoe.png.